### ISSUE TITLE
The function Input.MOUSE_MODE_CAPTURED, is failing

### ISSUE THREAD

**Issue by HectorVe** [godotengine/godot#62010]
### Godot version

3.4.4

### System information

Win10, OpenGL3

### Issue description

When the function is called, and I try rotate the Camera3D, the movement is incorrect of mouse

### Steps to reproduce

func _input(event):
	if event.is_action_pressed("ui_cancel"):
		mode = true if !mode else false
		
		if mode:
			Input.set_mouse_mode(Input.MOUSE_MODE_VISIBLE)
			get_tree().paused = true
		else:
			# THE BUGED FUNCTION IS THE NEXT
			Input.set_mouse_mode(Input.MOUSE_MODE_CAPTURED)
			get_tree().paused = false

### Minimal reproduction project

[corregir_error.zip](https://github.com/godotengine/godot/files/8895545/corregir_error.zip)

**Comment by HectorVe** [comment]
Here a file for test.
[corregir_error.zip](https://github.com/godotengine/godot/files/8895545/corregir_error.zip)

**Comment by HectorVe** [comment]
#61742

**Comment by BraqueDown** [comment]
NOTE: The code in the **Steps to reproduce** are not at all representative of the code in the MRP, and does not reflect the actual problem.
Having looked at the MRP, your issue is that ```InputEventMouseMotion.relative``` returns shorter distances when using ```MOUSE_MODE_CAPTURED``` than with ```MOUSE_MODE_VISIBLE```. (Making this a duplicate of #54818.)

There's a great explanation here: https://github.com/godotengine/godot/issues/56669#issuecomment-1009259577


TL;DR: ```MOUSE_MODE_CAPTURED``` causes ```InputEventMouseMotion``` to be triggered more often every frame, spreading the distance out in smaller parts.

**Comment by Calinou** [comment]
Duplicate of https://github.com/godotengine/godot/issues/54818.

@BraqueDown Thanks for the insightful explanation :slightly_smiling_face:

**Comment by HectorVe** [comment]
Thanks @BraqueDown ♥, I could fix the error with your help, and I see I'm very bad for find solutions. -_-

**Comment by BraqueDown** [comment]
No problem!
You can close this issue now.
